img/m/morganstanley.com.svg

```diff
@@ -0,0 +1,75 @@
+-34 -30 58 -29 182 1 245 36 73 106 100 172 65z"/>
+<path d="M1520 761 c-47 -10 -73 -25 -104 -62 l-26 -31 0 46 0 46 -45 0 -45 0
+0 -275 0 -275 45 0 45 0 0 146 c0 80 5 165 11 190 16 62 63 110 122 125 l47
+11 0 44 c0 24 -3 43 -7 43 -5 -1 -24 -4 -43 -8z"/>
+<path d="M1748 754 c-59 -18 -120 -84 -139 -151 -8 -27 -14 -85 -13 -131 1
+-108 35 -181 108 -230 42 -28 57 -32 111 -32 50 0 70 5 99 25 l36 24 0 -60 c0
+-55 -3 -64 -31 -90 -17 -15 -46 -31 -65 -35 -36 -7 -145 10 -176 27 -16 9 -18
+4 -18 -35 0 -44 1 -45 38 -55 48 -14 193 -14 229 0 15 5 41 22 57 37 54 51 56
+64 56 402 l0 310 -34 0 c-28 0 -35 -4 -40 -26 l-7 -27 -23 22 c-43 41 -109 50
+-188 25z m135 -75 c12 -6 32 -21 45 -32 21 -19 22 -28 22 -155 l0 -134 -33
+-33 c-66 -66 -159 -52 -201 30 -29 56 -30 202 -2 259 33 65 111 95 169 65z"/>
+<path d="M2305 761 c-16 -4 -44 -15 -61 -24 -32 -17 -84 -71 -84 -88 0 -5 16
+-17 36 -26 l36 -18 20 28 c63 87 207 71 211 -23 l2 -34 -60 -8 c-172 -22 -265
+-92 -265 -201 0 -120 100 -188 230 -157 28 7 60 20 72 30 22 17 23 17 51 -6
+18 -15 41 -24 63 -24 32 0 34 2 34 34 0 21 -6 36 -15 40 -13 4 -15 33 -15 175
+0 204 -9 238 -74 276 -45 27 -132 39 -181 26z m165 -330 c0 -61 -3 -71 -27
+-96 -85 -85 -203 -66 -203 32 0 77 45 104 223 132 4 0 7 -30 7 -68z"/>
+<path d="M2877 756 c-22 -8 -50 -24 -63 -37 l-24 -22 0 31 c0 32 -1 32 -50 32
+l-50 0 0 -275 0 -275 50 0 50 0 0 205 0 204 46 32 c25 17 63 33 84 36 32 5 43
+1 64 -20 l26 -26 0 -216 0 -215 45 0 45 0 0 228 c0 207 -2 230 -20 259 -36 59
+-130 86 -203 59z"/>
+<path d="M4382 761 c-46 -12 -104 -53 -124 -88 l-17 -30 36 -17 36 -17 28 30
+c31 35 66 51 109 51 61 0 106 -41 98 -91 -3 -21 -11 -24 -82 -35 -180 -28
+-273 -125 -236 -247 24 -81 105 -126 199 -112 52 8 76 19 109 48 18 17 22 18
+22 4 0 -23 39 -47 77 -47 31 0 33 2 33 34 0 18 -7 39 -15 46 -12 10 -15 44
+-15 190 0 167 -1 179 -22 210 -13 18 -40 42 -60 52 -42 22 -128 31 -176 19z
+m168 -334 c0 -83 -12 -102 -82 -132 -59 -24 -92 -20 -122 16 -68 80 3 161 159
+182 17 2 33 5 38 6 4 0 7 -32 7 -72z"/>
+<path d="M4965 759 c-16 -7 -45 -22 -62 -36 l-33 -24 0 30 c0 30 -1 31 -45 31
+l-45 0 0 -275 0 -275 45 0 45 0 0 198 0 198 28 26 c62 58 136 74 172 38 19
+-19 20 -33 20 -240 l0 -220 51 0 50 0 -3 229 c-3 214 -4 232 -24 257 -11 15
+-31 36 -45 46 -30 23 -113 31 -154 17z"/>
+<path d="M5710 758 c-88 -25 -146 -112 -157 -235 -7 -85 17 -184 55 -233 55
+-69 158 -99 265 -77 42 9 101 55 122 97 13 26 13 27 -22 44 l-35 17 -37 -40
+c-77 -83 -190 -59 -236 51 -8 21 -15 49 -15 63 l0 25 181 0 182 0 -5 78 c-5
+87 -31 148 -75 180 -55 39 -148 52 -223 30z m155 -90 c29 -23 55 -74 55 -108
+0 -19 -6 -20 -136 -20 l-136 0 7 33 c18 82 58 116 136 117 33 0 54 -6 74 -22z"/>
+<path d="M6060 752 c0 -5 41 -124 90 -265 50 -141 90 -264 90 -274 0 -10 -11
+-38 -23 -63 -27 -54 -76 -83 -124 -76 -32 5 -33 4 -33 -35 l0 -39 56 0 c31 0
+70 7 87 15 37 17 88 89 111 155 52 152 176 550 176 569 0 20 -4 22 -42 19
+l-41 -3 -53 -179 c-29 -98 -56 -186 -59 -195 -5 -10 -27 41 -61 139 -93 266
+-79 240 -130 240 -24 0 -44 -4 -44 -8z"/>
+</g>
+</svg>
```

Comment:
Oh wow. This is one of those corporate non-logo logos. I've minified the SVG a bit:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1000 147.9"><path d="M536.7 2.2a29 29 0 0 0-18.8 15.2 27.3 27.3 0 0 0 .6 26.7c4.2 7.2 14.2 13 37.4 21.4 16.8 6.1 23 13.7 20 24.5-2.6 10.1-15.2 16.2-28.1 13.2-9.1-2-14-6.4-21.6-19.2-1.4-2.3-2.5-2.5-8.2-1-3.5 1.1-6.4 2.4-6.4 3s2.3 5 5 9.5a41.4 41.4 0 0 0 45.5 20.4c20.3-3.4 29.6-13.6 29.6-32.8 0-17.2-6.8-23.2-40-35.6-15.8-5.9-21.6-11-21.6-19 0-11 12.7-18.2 26-14.6 7.7 2 13 6.3 16.2 13 2.6 5.6 4 6 12 4l4.1-.9-3-6.9A35.7 35.7 0 0 0 552.7.8a43.1 43.1 0 0 0-16 1.4zm283 50.2c0 35 .6 53.5 1.7 56 2.8 6 9.4 8 23.4 7.3 3.7-.1 4.2-.8 4.2-5.5 0-4.8-.5-5.4-4-5.4a20 20 0 0 1-7-1.6c-2.7-1.5-2.9-3.8-2.9-52.4V0h-15.4zM0 58.6v57h13.9l.3-47.3.4-47.5L26 58.6 40.4 106l2.9 9.7 5.7-.5 5.7-.4 28.4-94v47.3l.1 47.5h15.4V1.4L87 1.8l-11.7.5-12.8 42.2c-7 23.1-13 42.4-13.6 42.7A770 770 0 0 1 35.3 45l-13-42.7-11.2-.5L0 1.4zm611.7-40.9v13.1h-12.3v12.3h12.3V76c0 30.8.2 33 3 35.7 3.6 3.2 13.2 5.2 20 4.2 4.3-.7 4.8-1.3 4.8-5.9s-.5-5.2-4-5.2c-2.2 0-5.3-.8-7-1.6-2.8-1.5-3-3.5-3-30.8V43.1h14V30.8h-14V4.6h-13.8zM137.1 31.3A39.3 39.3 0 0 0 116.3 54a74 74 0 0 0-.7 36.2c5.5 17 17.4 25.7 34.8 25.6 22.3-.2 34.8-15.6 35-43.5a45 45 0 0 0-3.4-21 35.4 35.4 0 0 0-45-20.1zm21.5 12.6c9.8 5 15 26.7 10.4 42.5-6.3 21.5-27.2 24.4-37.1 5.3-4.6-9-4.5-28 .1-37.8 5.6-11.2 16.4-15.4 26.6-10zm75.6-13.2a24.9 24.9 0 0 0-16 9.5l-4 4.8V30.8h-13.9v84.8h13.9V93c0-12.4.8-25.5 1.7-29.3 2.4-9.6 9.7-17 18.8-19.3l7.2-1.7V36c0-3.7-.4-6.7-1-6.7zm35.1 1c-9 2.8-18.4 13-21.4 23.3a81.7 81.7 0 0 0-2 20.2c.2 16.6 5.4 27.9 16.7 35.4 6.4 4.4 8.7 5 17 5 7.8 0 10.9-.8 15.3-3.9l5.6-3.7v9.3c0 8.4-.5 9.8-4.8 13.8a26.7 26.7 0 0 1-10 5.4 66.7 66.7 0 0 1-27.1-4.1c-2.5-1.4-2.8-.7-2.8 5.4 0 6.7.1 6.9 5.8 8.4a97 97 0 0 0 35.3 0 30 30 0 0 0 8.8-5.7c8.3-7.8 8.6-9.8 8.6-62V30.9h-5.2c-4.3 0-5.4.6-6.2 4l-1 4.2-3.6-3.4c-6.6-6.3-16.8-7.7-29-3.9zm20.8 11.6a44.7 44.7 0 0 1 7 5c3.2 2.9 3.4 4.2 3.4 23.8v20.7l-5.1 5c-10.2 10.2-24.5 8-31-4.6a62 62 0 0 1-.3-39.9c5-10 17.1-14.6 26-10zm65-12.6a50 50 0 0 0-9.3 3.7c-5 2.6-13 10.9-13 13.5 0 .8 2.5 2.6 5.6 4l5.5 2.8 3.1-4.3c9.7-13.4 31.9-11 32.5 3.5l.3 5.3-9.2 1.2c-26.5 3.4-40.9 14.2-40.9 31 0 18.5 15.5 29 35.5 24.2 4.3-1.1 9.2-3.1 11-4.7 3.5-2.6 3.6-2.6 8 1 2.7 2.3 6.3 3.7 9.6 3.7 5 0 5.3-.3 5.3-5.3 0-3.2-1-5.5-2.3-6.1-2-.7-2.3-5.1-2.3-27 0-31.4-1.4-36.7-11.4-42.5a44 44 0 0 0-28-4zm25.5 50.8c0 9.4-.5 11-4.2 14.8-13 13.1-31.2 10.2-31.2-5 0-11.8 6.9-16 34.3-20.3.6 0 1 4.7 1 10.5zm62.7-50a31 31 0 0 0-9.7 5.6l-3.7 3.4v-4.8c0-4.9-.2-4.9-7.7-4.9h-7.7v84.8h15.4v-63l7-5c4-2.6 9.8-5 13-5.5 5-.8 6.7-.2 9.9 3l4 4v66.5h13.9V80.4c0-31.9-.3-35.4-3.1-39.9-5.6-9-20-13.2-31.3-9zm231.9-.8A36 36 0 0 0 656 44.2l-2.6 4.6 5.5 2.7 5.6 2.6 4.3-4.6a21.2 21.2 0 0 1 16.8-7.9c9.4 0 16.3 6.3 15 14-.4 3.3-1.6 3.7-12.6 5.4-27.7 4.3-42 19.3-36.3 38 3.7 12.6 16.2 19.5 30.6 17.3a27.7 27.7 0 0 0 16.8-7.4c2.8-2.6 3.4-2.7 3.4-.6 0 3.6 6 7.3 11.9 7.3 4.8 0 5-.4 5-5.3 0-2.8-1-6-2.2-7-1.9-1.6-2.3-6.8-2.3-29.3 0-25.8-.2-27.6-3.4-32.4a30.8 30.8 0 0 0-9.3-8 47.8 47.8 0 0 0-27.1-3zM701 82c0 12.8-1.9 15.7-12.7 20.4-9 3.7-14.1 3-18.7-2.5-10.5-12.3.4-24.8 24.4-28l5.9-1c.6 0 1 5 1 11.1zm64-51c-2.5 1-7 3.4-9.6 5.5l-5.1 3.7v-4.6c0-4.6-.2-4.8-7-4.8h-6.9v84.8h13.9v-61l4.3-4c9.5-9 21-11.5 26.5-6 3 3 3 5.2 3 37v34h15.6l-.4-35.3c-.5-33-.6-35.8-3.7-39.6a37.9 37.9 0 0 0-7-7.1c-4.6-3.6-17.4-4.8-23.7-2.6zm114.7.1c-13.5 3.9-22.5 17.3-24.2 36.2a57.8 57.8 0 0 0 8.5 36c8.5 10.6 24.3 15.2 40.8 11.8a33.2 33.2 0 0 0 18.8-15c2-4 2-4.1-3.4-6.7l-5.4-2.6-5.7 6.1c-11.8 12.8-29.2 9.1-36.3-7.8a34.9 34.9 0 0 1-2.3-9.7v-3.9h55.9l-.8-12C925 50 921 40.7 914.2 35.7a43 43 0 0 0-34.4-4.6zm24 13.9a25.3 25.3 0 0 1 8.5 16.6c0 3-1 3.1-21 3.1h-21l1.1-5c2.8-12.7 9-18 21-18.1 5 0 8.3 1 11.4 3.4zm30-13c0 .8 6.4 19.2 14 40.9a557.5 557.5 0 0 1 13.8 42.2 35 35 0 0 1-3.6 9.7c-4.1 8.3-11.7 12.8-19 11.7-5-.7-5.2-.6-5.2 5.4v6h8.7a39 39 0 0 0 13.4-2.3c5.7-2.6 13.5-13.7 17-23.9 8-23.4 27.2-84.7 27.2-87.6 0-3.1-.6-3.4-6.5-3l-6.3.5-8.2 27.6-9 30c-.8 1.6-4.2-6.3-9.5-21.4-14.3-41-12.1-37-20-37-3.7 0-6.8.6-6.8 1.2z"/></svg>
```